src/xenia/gpu/ucode.h

```diff
@@ -145,7 +145,7 @@ enum class AllocType : uint32_t {
 };
 
 // Instruction data for ControlFlowOpcode::kExec and kExecEnd.
-struct ControlFlowExecInstruction {
+XEPACKEDSTRUCT(ControlFlowExecInstruction, {
```

Comment:
Alignment of 4, not 1, is safe in these structures — no need to make them `XEPACKEDSTRUCT`. In most cases, `XEPACKEDSTRUCT` can be removed from bitfields overall.

Reply:
The struct is used in a packed union and therefore GCC complains:
`ignoring packed attribute because of unpacked non-POD field`

So I made them packed but encapsulated the bit field in another struct. I assumed the packed attribute is not inherited by anonymous structs, is this wrong?

Reply:
It's safe to remove `XEPACKEDSTRUCT` and `XEPACKEDUNION` from everywhere in `ucode.h`. I'm not sure about the inner structs, probably yes, but it's weird that it's used at all in this case then.